parts/common/components.json

```diff
@@ -1462,6 +1462,12 @@
                 "renovateTag": "OCI_registry=https://mcr.microsoft.com, name=oss/binaries/kubernetes/kubernetes-node",
                 "latestVersion": "v1.33.4",
                 "previousLatestVersion": "v1.33.3"
+              },
+              {
+                "k8sVersion": "1.34",
+                "renovateTag": "OCI_registry=https://mcr.microsoft.com, name=oss/binaries/kubernetes/kubernetes-node",
+                "latestVersion": "v1.34.0",
+                "previousLatestVersion": "v1.34.0"
```

Comment:
Did you intend to change this?

Reply:
I needed something to trigger the VHD build and this was a good start. We also do need to cache the 1.34 k8s binaries as this is about to go public preview.

Reply:
I needed something to trigger the VHD build and this was a good start. We also do need to cache the 1.34 k8s binaries as this is about to go public preview.